framework/common/encryption.cpp

```diff
@@ -441,22 +442,26 @@ void wps_calculate_keys(const diffie_hellman &dh, const uint8_t *remote_pubkey,
     // "SHA256_MAC_LEN" bytes long (ie. 're_len' must always be "SHA256_MAC_LEN",
     // even if it is an input argument)
     //
-    union {
-        uint32_t i;
-        uint8_t buf[4];
-    } kdf_iter, kdf_key_length;
+    // union {
+    //     uint32_t i;
+    //     uint8_t buf[4];
+    // } kdf_iter, kdf_key_length;
+
```

Comment:
If this union is not used anymore, it should be removed, not commented out.

Reply:
True, removed.